mitosheet/src/mito/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -80,52 +114,13 @@ const GraphSidebar = (props: {
         }
     }, [props.graphSidebarTab])
 
-    /* 
-        When graphUpdatedNumber is updated, we send a new getGraphMessage with the current graphParams
-        in order to update the graphDataDict. We only increment graphUpdatedNumber when the user updates the params.
-
-        We use this method instead of using a useEffect on the graphParams because the graphParams update when we don't want
-        to sendGraphMessage, ie: during an Undo. 
-    */
-    const [graphUpdatedNumber, setGraphUpdatedNumber] = useState(0)
-
     // We log if plotly is not defined
     useEffect(() => {
         if (!(window as any).Plotly) {
             void props.mitoAPI.log('plotly_define_failed');
         }
     }, [])
 
-    // Whenever the graph is resized, we update it (so it resizes as well)
-    useEffectOnResizeElement(() => {
-        setGraphUpdatedNumber(old => old + 1)
-    }, [], props.mitoContainerRef, '#mito-center-content-container')
```

Comment:
I think we need to still run this effect -- it makes sure graphs are sized correctly. 

I think you can just change `setGraphUpdatedNumber(old => old + 1)` to like `setGraphParams` to a copy (make sure to test resizing).